plugins/glustershd/rest.go

```diff
@@ -1,17 +1,22 @@
 package glustershd
 
 import (
+	"encoding/xml"
 	"net/http"
+	"path"
 
 	"github.com/gluster/glusterd2/glusterd2/gdctx"
 	restutils "github.com/gluster/glusterd2/glusterd2/servers/rest/utils"
 	"github.com/gluster/glusterd2/glusterd2/transaction"
 	"github.com/gluster/glusterd2/glusterd2/volume"
 	"github.com/gluster/glusterd2/pkg/errors"
 
+	glustershdapi "github.com/gluster/glusterd2/plugins/glustershd/api"
```

Comment:
This import should be part of the above block as it's also an internal glusterd2 package.